library.js

```diff
@@ -1,7 +1,158 @@
-class Book{
-    constructor(title,author,pages){
-        this.title = title;
-        this.author = author;
-        this.pages = pages;
+
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const pages = document.getElementById('pages');
+const status = document.getElementById('status-btn');
+class Book {
+  constructor(title, author, pages) {
+    this.title = title;
+    this.author = author;
+    this.pages = pages;
+    this.status = false;
+  }
+}
+
+// let clicked = false;
+// function toggle() {
+//   if (!clicked) {
+//     clicked = true;
+//     document.getElementById('toggle-btn').innerHTML = 'unread';
+//   } else {
+//     clicked = false;
+//     document.getElementById('toggle-btn').innerHTML = 'read';
+//   }
+// }
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

Reply:
I have no coding partner, I work alone in this project.